src/object.d

```diff
@@ -155,15 +164,269 @@ auto opEquals(Object lhs, Object rhs)
     return lhs.opEquals(rhs) && rhs.opEquals(lhs);
```

Comment:
Okay. It now checks that both of the types being compared have a member named `opEquals` in addition to checking that the result of calling it is `bool` like it did before.